data/util/NumericShaperUtility.js

```diff
@@ -0,0 +1,294 @@
+define(
```

Comment:
Why was this put in `data/util`? `dojo/data` stores are deprecated and it looks like this isn't really used for `dojo/data` anyway. Should this be in `cldr`?